Source/Resources/Content/AadIam/FeatureSettings/Configuration.json

```diff
@@ -1,6 +1,7 @@
 {
   "convergedUXV2FeatureValue": 2,
   "convergedUXV2GroupId": null,
+  "description":"What is it: Allows Azure updates to be applied first to a specified group of users before being applied to all users in the tenant. ",
```

Comment:
**What does this do?**
Allows Azure updates to be applied first to a specified group of users before being applied to all users in the tenant.

**Why should you use this?**
If you want to test Azure updates with a subset of users in your environment. The specified users will receive Azure updates before other users.

**What is the end user impact?**
Specified users will be able to test Azure updates before other users.

**Learn more**
N/A